src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/TagHelperDescriptorJsonConverter.cs

```diff
@@ -12,8 +13,14 @@ internal class TagHelperDescriptorJsonConverter : JsonConverter
     {
         public static readonly TagHelperDescriptorJsonConverter Instance = new TagHelperDescriptorJsonConverter();
 
+        private static readonly WeakReferenceTable weakReferenceTable = WeakReferenceTable.Instance;
+
         public static bool DisableCachingForTesting { private get; set; } = false;
 
+        internal TagHelperDescriptorJsonConverter()
```

Comment:
Why this?

Reply:
Good point. I was originally going to make an empty private constructor to keep people from using anything but the instance, but now that we have the table that would violate statelessness (though hypothetically should not affect results) in the unit tests.